pkgs/development/python-modules/runstats/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  setuptools,
+  cython,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  pytest-cov-stub,
+  pytest-xdist,
+}:
+
+buildPythonPackage rec {
+  pname = "runstats";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "grantjenks";
+    repo = "python-runstats";
+    tag = "v${version}";
+    hash = "sha256-YF6S5w/ccWM08nl9inWGbaLKJ8/ivW6c7A9Ny20fldU=";
+  };
+
+  build-system = [
+    setuptools
+    cython
+  ];
+
+  nativeCheckInputs = [
+    pytest-cov-stub
+    pytest-xdist
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "runstats" ];
+
+  meta = {
+    description = "Python module for computing statistics and regression in a single pass.";
```

Comment:
`meta.description` should not end with a period (see [docs](https://github.com/NixOS/nixpkgs/blob/9a0f387ecc4a7958040ffa37ea6d6651bd751604/pkgs/README.md#meta-attributes))

```suggestion
    description = "Python module for computing statistics and regression in a single pass";
```

Reply:
Definitely doable.  i am going to force push a new commit right away.